### PR TITLE
Create multiple tags if more than one crate in a repository

### LINKED ISSUE #49: Create multiple tags if more than one crate in a repository
In cairo, sourceviez and sys repositories, multiple crates are in one repository, which is problematic for the tagging. I think we should put sys crates behind a tag like `crate-sys-version` and don't change anything for "main" crate.

Original discussion can be found here: https://github.com/gtk-rs/sys/pull/109

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #49.

Still needs to be tested though but I think I'm close to the end...

### CODE REVIEW

File: src/release.py [hunk 1]
```diff
@@ -477,48 +478,22 @@ def build_blog_post(repositories, temp_dir, token):
...
-                version = common
-        if version is None:
-            write_error('Something impossible happened for "{}": no version can be tagged...'
-                        .format(repository))
-            input('If you think you can do better, go ahead! (In "{}") Then press ENTER to continue'
-                  .format(join(temp_dir, repository)))
-            return
-    write_msg('==> Creating new tag "{}" for repository "{}"...'.format(version, repository))
-    create_tag_and_push(version, repository, temp_dir)
+                tag_name = '{}-{}'.format(crate['crate'], CRATES_VERSION[crate['crate']])
```
Comment (EPashkin) [inline src/release.py]: why not `tag_name = '{}-{}'.format(crate['crate'], tag_name)` ?

Comment (EPashkin) [reply src/release.py]: :wink:

**Comment by EPashkin** [comment]
Thanks

File: src/release.py (hunk 1 above)
Comment (GuillaumeGomez) [reply src/release.py]: But thanks in any case, I'll update this evening. :smiley:

Comment (EPashkin) [reply src/release.py]: It was good as is it just unneeded nit

Comment (GuillaumeGomez) [reply src/release.py]: I think your suggestion is better than the current code so I'll follow it.

**Comment by GuillaumeGomez** [comment]
I updated it (and ran it). Seems to be working as expected, we now have some nice tags for every crates in every repository.

**Comment by EPashkin** [comment]
Cairo need manual 0.7.0 replace with 0.5.0

**Comment by GuillaumeGomez** [comment]
Oh indeed! And I think that I need to remove some tags from `sys` as well...

**Comment by EPashkin** [comment]
Oh, yes, I missed 0.8.0 in sys, it on other commit

**Comment by GuillaumeGomez** [comment]
And same for `sourceview` I assume. ;)

**Comment by GuillaumeGomez** [comment]
Ok, I removed the extra tags. Is this PR good for you or is there anything else I need to change?

**Comment by EPashkin** [comment]
All looks good for me

**Comment by GuillaumeGomez** [comment]
Then I merge. Thanks for reviewing! :)

**Pull request merged**